requirements.txt

```
@@ -5,8 +5,10 @@ dataclasses
 
 black[jupyter]
 clang-format==14
```

Just realized that `==14` only installs `14.0.0` rather than the highest available subversion of 14 😅 
```suggestion
clang-format==14.*
```